tutorials/general/t8_tutorial_build_cmesh_main.cxx

```diff
@@ -0,0 +1,35 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element types in parallel.
+
+  Copyright (C) 2023 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+/* This file is part of the step6 example of t8code.
+ * Here, we only wrap the main function. See t8_step8_user_defined_mesh.cxx for the documentation of
+ * the step8 example.
+ */
+
+#include <tutorials/general/t8_tutorial_build_cmesh.h>
+
+int
+main (int argc, char **argv)
+{
+  return t8_step8_main (argc, argv);

```

Comment:
```suggestion
  return t8_tutorial_build_cmesh_main(argc, argv);
```

Reply:
And the function call to the tutorial's "main-function" has to be updated here accordingly